docs/authentication/enterprise-connections/overview.mdx

```diff
@@ -72,13 +72,3 @@ The users will not be deleted, so your app will not break. However, they will ne
 ### Does Clerk support IdP-initiated SSO?
 
 Yes, for SAML only. Clerk supports both Service Provider-initiated (SP-initiated) and Identity Provider-initiated (IdP-initiated) SSO flows. For more information, see the [authentication flows guide](/docs/authentication/enterprise-connections/authentication-flows).
-
-### How much does it cost?
-
-For development instances, Enterprise connections are always free but limited to a maximum of 25.
-
-Production instances require the Pro plan and the Enhanced Authentication Add-on. Please see [pricing](/pricing){{ target: '_blank' }} for more information.
-
-### Can I get a bulk discount?
-
-Yes, [contact support](/contact/support){{ target: '_blank' }} to work out a custom plan.
```

Comment:
We can delete the "Can I get a bulk discount?" section, but everything in the "How much does it cost?" section is still correct

Reply:
put it back!! [14c7f83](https://github.com/clerk/clerk-docs/pull/1802/commits/14c7f83ab36f40094740821e74c06691216729ef)